test_notebooks.py

```diff
@@ -84,16 +84,15 @@ def reduce_notebook_load(path):
 
 Dir_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "notebooks")
 
-@pytest.mark.parametrize("notebook", glob(os.path.join(Dir_path, "basic*.ipynb")) +
-                         [os.path.join(Dir_path, "introduction_python.ipynb"),
-                          os.path.join(Dir_path, "introduction_dataset.ipynb"),
-                          os.path.join(Dir_path, "introduction_quickstart.ipynb"),
-                          os.path.join(Dir_path, "introduction_showcase.ipynb")] +
+@pytest.mark.parametrize("notebook",
+                         glob(os.path.join(Dir_path, "introduction_*.ipynb")) +
```

Comment:
I thought I was getting some errors when I was testing the intro notebook about jupyter (or others), but if it works that's great :)

Reply:
Yes, you're right. But I've added the keywords `TAB` and `CRASH` to the error catch function in [line33](https://github.com/miykael/nipype_tutorial/pull/72/files#diff-a6a88b4925c00fdd2e781a0510255b48R33).

Now, if you add in a cell that will crash the comment `#This will CRASH` or `#Try TAB for completion`, it will catch the crash and circleci can continue :-)

Reply:
oh, nice :)

Reply:
btw. for the same reason I included `try/except` blocks in the main nipype tutorials. Feel free to remove them, the good thing about the `try/except` blocks is that I tried to specify the error, to be sure that we're catching the right one.

Reply:
Yes, you're right. Your `try/except` are very good. To catch the right errors, and also to show the people (and me) how to handle errors the right way.

But at least in the jupyter_notebook with the TAB completion example, a `try/except` will not make sense.

Reply:
yes, totally agree. It was a great idea